db/migrations/20161018054000_buzzerdb.sql

```diff
@@ -14,16 +22,8 @@ CREATE TABLE ActiveParties (
   time_seated timestamp,
   phone_ahead boolean NOT NULL,
   wait_time_expected int,
-  wait_time_calculated int
-);
-
-CREATE TABLE Buzzers (
-  id serial PRIMARY KEY,
-  restaurant_id int REFERENCES Restaurants(id),
-  buzzer_name VARCHAR(45) NOT NULL,
-  last_heartbeat timestamp,
-  is_active boolean NOT NULL,
-  active_parties_id int REFERENCES ActiveParties(id)
+  wait_time_calculated int,
+  buzzers_id int REFERENCES Buzzers(id)
```

Comment:
Can you make this buzzer_id? I think you had it as singular in the model declaration